cfme/fixtures/v2v_fixtures.py

```diff
@@ -299,6 +309,24 @@ def infra_mapping_default_data(source_provider, provider):
     return infra_mapping_data
 
 
+@pytest.fixture(scope="function")
+def mapping_data_vm_obj_mini(request, appliance, source_provider, provider):
+    """Fixture which provides minimal mapping data, vm and map object for migration plan"""
+    infra_mapping_data = infra_mapping_default_data(source_provider, provider)
+    vm_obj = get_vm(request, appliance, source_provider, template=rhel7_minimal)
+
+    infrastructure_mapping_collection = appliance.collections.v2v_infra_mappings
+    mapping = infrastructure_mapping_collection.create(**infra_mapping_data)
+
+    @request.addfinalizer
+    def _cleanup():
+        vm_obj.cleanup_on_provider()
+        infrastructure_mapping_collection.delete(mapping)
+
+    return FormDataVmObjMapObj(
+        infra_mapping_data=infra_mapping_data, vm_list=[vm_obj], map_obj=mapping)
```

Comment:
Can we get mapping name from mapping_data_vm_obj_mini.infra_mapping_data.name ?
If yes then we will not need the mapping object

Reply:
Yes, object was only to retrieve name so good to remove it.